src/helpers/firebaseConfig.js

```diff
@@ -1,4 +1,8 @@
-export default {
+import firebase from "firebase";
+import "firebase/firestore";
+
+// firebase init goes here
+const config = {
   apiKey: "AIzaSyAdeWHYbSj2iErECQTncQLrz9WdfbuiCsQ",
```

Comment:
Are you sure this `apiKey` is safe to expose to the public?

Reply:
Yes, these configurations are required in order to connect to firebase, it should only act as an identifier. This means we don't need to set up an intermediate backend server to relay the request. See [here](https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public)